manta-js/examples/asset-webpack-ts/yarn.lock

```diff
@@ -5435,6 +5450,11 @@ safe-buffer@5.2.1, safe-buffer@>=5.1.0, safe-buffer@^5.0.1, safe-buffer@^5.1.0,
   resolved "https://registry.npmmirror.com/safer-buffer/-/safer-buffer-2.1.2.tgz#44fa161b0187b9549dd84bb91802f9bd8385cd6a"
   integrity sha512-YZo3K82SD7Riyi0E1EQPojLz7kpepnSQI9IyPbHHg1XXXevb5dJI7tpyN2ADxGcQbHG7vcyRHk0cbwqcQriUtg==
 
+scale-codec@^0.10.0:
+  version "0.10.2"
```

Comment:
I think it's fine to bump `scale-codec` directly to `^0.10.2`.

Reply:
Yeah me too